modules/serverless-functions-podman.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-functions-setup.adoc
+
+[id="serverless-functions-podman_{context}"]
+= Using podman
+
+If you are using podman, you must run the following commands before getting started with {FunctionsProductShortName}:
```

Comment:
s/{FunctionsProductShortName}/{FunctionsProductName}

since this is the first reference of the product name in this assembly